integration/fs_test.go

```diff
@@ -40,6 +42,24 @@ func TestFilesystem(t *testing.T) {
 			},
 			golden: "testdata/gomod.json.golden",
 		},
+		{
+			name: "gomod with skip files",
+			args: args{
+				securityChecks: "vuln",
+				input:          "testdata/fixtures/fs/gomod",
+				skipFiles:      []string{"/testdata/fixtures/fs/gomod/submod2/go.mod"},
+			},
+			golden: "testdata/gomod-skip-files.json.golden",
+		},
+		{
+			name: "gomod with skip dirs",
+			args: args{
+				securityChecks: "vuln",
+				input:          "testdata/fixtures/fs/gomod",
+				skipDirs:       []string{"/testdata/fixtures/fs/gomod/submod2"},
+			},
+			golden: "testdata/gomod-skip-dirs.json.golden",
```

Comment:
Can we use the same golden file as the test for skip files?

Reply:
fixed.